.spelling

```diff
@@ -100,6 +100,7 @@ CECs
 center
 centers
 Ceph
+ceph
```

Comment:
Any time we refer to Ceph the product it should be capitalized. If we're talking about the command, we should be enclosing it in backticks, which will bypass the spell checker.
```suggestion
```